# Задача 2

Для умов опалення приватної оселі через циркуляцію нагрітої води в замкненому контурі,
порівняти витрати на енергоносії в умовах існуючих цін (з'ясувати) при використанні природного газу та електрики.
Для інженерного розрахунку вважати, що склад природного газу 100% метан, а ККД процесу нагріву води в обох випадках однаковий.
Теплоту згоряння метану взяти в довіднику.

## Вступ

Питання енергоефективності та оптимального вибору енергоносіїв для опалення приватних осель є надзвичайно актуальним у контексті постійних змін на енергетичному ринку України. Дане дослідження присвячене порівняльному аналізу економічної доцільності використання природного газу та електроенергії для опалення приватної оселі в умовах існуючих цін на енергоносії. Особлива увага приділяється розрахунку вартості генерації однакової кількості теплової енергії за допомогою різних енергоносіїв з урахуванням їхніх теплотворних характеристик та ефективності систем опалення.


## Теоретичні основи

### Термодинамічні аспекти процесів опалення

Для коректного порівняння енергоносіїв необхідно враховувати їхні фізичні властивості, зокрема теплоту згоряння. Теплота згоряння – це кількість теплової енергії, що виділяється при повному згорянні одиниці маси або об'єму палива за нормальних умов.

Для природного газу теплота згоряння зазвичай вимірюється в МДж/м³. Відповідно до наданих даних, теплота згоряння чистого метану становить:

$$Q_{\text{метан}} = 50.03 \text{ МДж/м³}$$

Для природного газу, який містить домішки інших компонентів, теплота згоряння нижча і становить:

$$Q_{\text{прир.газ}} = 33.50 \text{ МДж/м³}$$

In [22]:
# Теплота згоряння
METHANE_HEAT_VALUE = 50.03  # МДж/м³ (для чистого метану)
NATURAL_GAS_HEAT_VALUE = 33.50  # МДж/м³ (для природного газу)

### Коефіцієнт корисної дії (ККД)

Важливою характеристикою системи опалення є коефіцієнт корисної дії (ККД), який визначає частку енергії, що ефективно перетворюється на корисне тепло. Для різних типів опалювальних систем ККД може суттєво відрізнятися:

$$\eta = \frac{Q_{\text{корисне}}}{Q_{\text{загальне}}}$$

де $\eta$ – коефіцієнт корисної дії системи опалення, $Q_{\text{корисне}}$ – корисна теплова енергія, $Q_{\text{загальне}}$ – загальна затрачена енергія.

У нашому дослідженні прийнято такі значення ККД:
- ККД газового котла: $\eta_{\text{газ}} = 0.9$ (90%)
- ККД електричного котла: $\eta_{\text{електр}} = 0.9$ (90%)
- ККД системи в цілому (з урахуванням втрат у трубах тощо): $\eta_{\text{система}} = 0.9$ (90%)

In [23]:
# ККД
GAS_BOILER_EFFICIENCY = 0.9  # 90% ККД газового котла
ELECTRIC_BOILER_EFFICIENCY = 0.9  # 90% ККД електричного котла
SYSTEM_EFFICIENCY = 0.9  # 90% ККД системи опалення (втрати у трубах тощо)

### Перетворення енергетичних одиниць

Для коректного порівняння різних енергоносіїв необхідно перевести всі енергетичні величини до єдиної системи одиниць. У даному дослідженні використовується коефіцієнт перетворення мегаджоулів у кіловат-години:

$$1 \text{ МДж} = 0.2778 \text{ кВт·год}$$

In [24]:
# Константи для перетворення одиниць
MJ_TO_KWH = 0.2778  # 1 МДж = 0.2778 кВт·год

### Тепловий насос та коефіцієнт перетворення енергії (COP)

Тепловий насос – це пристрій, який переносить теплову енергію від нижчого потенціалу до вищого за допомогою затрати зовнішньої роботи. Ефективність теплового насоса характеризується коефіцієнтом перетворення енергії (COP - Coefficient of Performance):

$$COP = \frac{Q_{\text{теплова}}}{W_{\text{електрична}}}$$

де $Q_{\text{теплова}}$ – отримана теплова енергія, $W_{\text{електрична}}$ – затрачена електрична енергія.

Для сучасних теплових насосів COP може варіюватися від 2.5 до 5.0 і вище, залежно від типу теплового насоса та умов експлуатації.

## Розв'язання

### Вхідні дані

Для проведення порівняльного аналізу використано наступні вхідні дані:

1. Тарифи на енергоносії:
   - Електроенергія: 4.32 грн/кВт·год
   - Природний газ: 7.96 грн/м³

2. Теплотворна здатність палива:
   - Метану: 50.03 МДж/м³
   - Природного газу: 33.50 МДж/м³

3. Коефіцієнти корисної дії:
   - Газового котла: 90%
   - Електричного котла: 90%
   - Системи опалення в цілому: 90%

4. Потреба в тепловій енергії: 1000 кВт·год

In [25]:
# Константи для розрахунків (актуальні дані 2025 року)
ELECTRICITY_PRICE = 4.32  # грн/кВт·год
GAS_PRICE = 7.96  # грн/м³

# Теплова потреба будинку для розрахунку
HEAT_DEMAND = 1000  # кВт·год тепла за місяць

### Методика розрахунку

Розрахунок витрат на різні енергоносії здійснювався за наступним алгоритмом:

1. Обчислення енергетичного еквіваленту 1 м³ газу (в кВт·год):
   $$E_{\text{газ/м³}} = Q_{\text{газ}} \cdot 0.2778$$

2. Визначення ефективної енергії з урахуванням ККД:
   $$E_{\text{ефект}} = E_{\text{газ/м³}} \cdot \eta_{\text{газ}} \cdot \eta_{\text{система}}$$

3. Розрахунок необхідного об'єму газу для задоволення потреби в теплі:
   $$V_{\text{газ}} = \frac{E_{\text{потреба}}}{E_{\text{ефект}}}$$

4. Обчислення вартості газу:
   $$C_{\text{газ}} = V_{\text{газ}} \cdot T_{\text{газ}}$$

5. Розрахунок необхідної кількості електроенергії з урахуванням ККД:
   $$E_{\text{електр}} = \frac{E_{\text{потреба}}}{\eta_{\text{електр}} \cdot \eta_{\text{система}}}$$

6. Обчислення вартості електроенергії:
   $$C_{\text{електр}} = E_{\text{електр}} \cdot T_{\text{електр}}$$

7. Порівняння витрат та визначення відносної ефективності:
   $$\text{Коефіцієнт} = \frac{C_{\text{електр}}}{C_{\text{газ}}}$$

8. Для теплового насоса додатково розраховується витрата електроенергії з урахуванням COP:
   $$E_{\text{тн}} = \frac{E_{\text{потреба}}}{COP \cdot \eta_{\text{система}}}$$

9. Визначення критичного значення COP, при якому електричне опалення з тепловим насосом стає економічно вигіднішим за газове:
   $$COP_{\text{крит}} = \frac{E_{\text{електр}} \cdot T_{\text{електр}}}{C_{\text{газ}}}$$

### Розрахунок енергетичних характеристик метану

Розрахуємо енергетичний еквівалент 1 м³ метану:

In [26]:
# Розрахунок енергії від 1 м³ метану
methane_energy_per_m3 = METHANE_HEAT_VALUE * MJ_TO_KWH  # кВт·год/м³
print(f"Енергія від 1 м³ метану: {methane_energy_per_m3:.2f} кВт·год")

Енергія від 1 м³ метану: 13.90 кВт·год


$$E_{\text{метан/м³}} = Q_{\text{метан}} \cdot 0.2778 = 50.03 \text{ МДж/м³} \cdot 0.2778 \text{ кВт·год/МДж} = 13.90 \text{ кВт·год/м³}$$

З урахуванням ККД газового котла та системи опалення:

In [27]:
methane_energy_with_efficiency = methane_energy_per_m3 * GAS_BOILER_EFFICIENCY * SYSTEM_EFFICIENCY  # кВт·год/м³ з урахуванням ККД
print(f"Енергія від 1 м³ метану з урахуванням ККД: {methane_energy_with_efficiency:.2f} кВт·год")

Енергія від 1 м³ метану з урахуванням ККД: 11.26 кВт·год


$$E_{\text{метан/ефект}} = E_{\text{газ/м³}} \cdot \eta_{\text{газ}} \cdot \eta_{\text{система}} = 13.90 \text{ кВт·год/м³} \cdot 0.9 \cdot 0.9 = 11.26 \text{ кВт·год/м³}$$

Для забезпечення 1000 кВт·год теплової енергії потрібно:

In [28]:
# Розрахунок кількості газу для забезпечення теплової потреби
methane_volume_needed = HEAT_DEMAND / methane_energy_with_efficiency  # м³
print(f"Кількість метану для забезпечення {HEAT_DEMAND} кВт·год тепла: {methane_volume_needed:.2f} м³")

Кількість метану для забезпечення 1000 кВт·год тепла: 88.83 м³


$$V_{\text{метан}} = \frac{E_{\text{потреба}}}{E_{\text{ефект}}} = \frac{1000 \text{ кВт·год}}{11.26 \text{ кВт·год/м³}} = 88.83 \text{ м³}$$

Вартість метану:

In [29]:
# Розрахунок витрат на газ
methane_cost = methane_volume_needed * GAS_PRICE  # грн
print(f"Витрати на метан: {methane_cost:.2f} грн")

Витрати на метан: 707.07 грн


$$C_{\text{метан}} = V_{\text{метан}} \cdot T_{\text{газ}} = 88.83 \text{ м³} \cdot 7.96 \text{ грн/м³} = 707.07 \text{ грн}$$